app/api/helpers/ticketing.py

```diff
@@ -89,7 +89,7 @@ def charge_stripe_order_payment(order, token_id):
             save_to_db(order)
 
             invoice_id = order.get_invoice_number()
-            order_url = make_fe_url(path="/{identifier}/view/".format(identifier=order.identifier))
+            order_url = make_frontend_url(path="/{identifier}/view/".format(identifier=order.identifier))
```

Comment:
Don't think this i needed since you have commented out the lines requiring this. please verify.